app.js

```diff
@@ -74,15 +77,15 @@ function countriesQuestion() {
   }
   return guessCountry;
 }
-var rightCountry = countriesQuestion();
+var guessCountry = countriesQuestion();
```

Comment:
The variable above is inside the function. I renamed the variable outside the function to clear up confusion